eng/common/templates-official/job/publish-build-assets.yml

```diff
@@ -60,8 +60,8 @@ jobs:
       os: windows
     # If it's not devdiv, it's dnceng
     ${{ if ne(variables['System.TeamProject'], 'DevDiv') }}:
-      name: $(DncEngInternalBuildPool)
-      image: 1es-windows-2022-pt
+      name: NetCore1ESPool-Publishing-Internal
+      image: windows.vs2019.amd64
```

Comment:
Don't they have 2022 image?

Reply:
The source for this change is https://github.com/dotnet/arcade/pull/14663.

Is this important?  As long as CI builds keep working, it seems OK.  It has nothing to do with the .NET SDK version we use to build Sign CLI.  The Arcade team changed agent pools.  Maybe they're planning to update images within the pool.  A benefit of Arcade is that the Arcade team manages these details, so we don't have to.